csi/volumeutils.py

```diff
@@ -1218,7 +1148,10 @@ def mount_glusterfs(volume, mountpoint, storage_options="", is_client=False):
                 raise err
         return mountpoint
 
-    generate_client_volfile(volname)
+    # Todo:
```

Comment:
Remove this "TODO" also. Let's remove options completely from Storage class or PV create/mount flow. Let's keep only as Storage pool configuration.

Reply:
Also remove the lines with Volfile modifications. That way we can have uniformity once NFS mount also introduced. Options can only be configured for Storage pools via operator.

Reply:
Removed volume_options from mount flow.